util/Setup/Program.cs

```diff
@@ -196,14 +197,21 @@ private static bool ValidateInstallation()
     {
         var installationId = string.Empty;
         var installationKey = string.Empty;
+        CloudRegion cloudRegion;
 
         if (_context.Parameters.ContainsKey("install-id"))
         {
             installationId = _context.Parameters["install-id"].ToLowerInvariant();
         }
         else
         {
-            installationId = Helpers.ReadInput("Enter your installation id (get at https://bitwarden.com/host)");
+            var prompt = "Enter your installation id (get at https://bitwarden.com/host)";
+            installationId = Helpers.ReadInput(prompt);
+            while (installationId == string.Empty)
```

Comment:
It's generally safer to do `string.IsNullOrEmpty()` and same for all the other `== string.Empty` checks.

Reply:
Updated all of them to `string.IsNullOrEmpty()`.